backend/src/routes/teamRoutes.ts

```diff
@@ -17,5 +19,7 @@ teamRoutes.put("/:id", updateTeam);
 teamRoutes.delete("/:id", deleteTeam);
 teamRoutes.delete("/:id/member", deleteMember);
 teamRoutes.post("/:id/member", addMembers);
+teamRoutes.post("/:id/member/add-expense-data", addExpenseAccountData);
+teamRoutes.get("/:id/member", getExpenseAccountData)
```

Comment:
```suggestion
teamRoutes.post("/:id/add-expense-data", addExpenseAccountData);
teamRoutes.get("/:id/get-expense-data", getExpenseAccountData)
```

Reply:
This structure make the routing more clear

Reply:
it should be a noun not a verb.
```
teamRoutes.get("/:id/expense-data", getExpenseAccountData)
```

even better if we have a memberRoutes so it looks like

`/members/:id/expense-data`

Reply:
https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design/#h2-2eb87cd076da0

Reply:
Yes sorry, it's should be a noun Thanks.